examples/GUI/GUIGlobe.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GUIGlobe
+ * Description: Classe pour créer un menu.
+ */
+
+
+function GUIGlobe() {
```

Comment:
I don't really see the point of `GUIGlobe`.
What are the benefits of introduce this class over using `dat.gui` directly?
From my point of view, there's no need to introduce a new class if you just a new a few simple helpers methods in a single place...